.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,11 @@
+### What was wrong?
+
```

Comment:
Same, a PR could be a feature ;)

Reply:
And `What's your pull request in response to?`

Reply:
We could get the questions from moby repo:


**- What I did**

**- How I did it**

**- How to verify it**

**- Description for the changelog**